### PR TITLE
customers#45 Enable gzip compression

### PR DESCRIPTION
**Pull request by oubchid**
Module ngx_http_gzip_static_module already added here https://github.com/linagora/esn-frontend-contacts/commit/8965e2315f84808def48b75997b376715b2b4575

### CODE REVIEW

**Review by tanandy — DISMISSED** [review]

**Comment by rezk2ll** [comment]
missing nginx conf?

**Comment by tanandy** [comment]
> missing nginx conf?

see the comment from @oubchid it has been set in a previous commit

**Review by rezk2ll — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "chai-datetime": "1.5.0",
     "chai-shallow-deep-equal": "1.4.6",
     "clean-webpack-plugin": "^3.0.0",
+    "compression-webpack-plugin": "^8.0.1",
```
Comment (tanandy) [inline package.json:34]: https://github.com/linagora/esn-frontend-inbox/blob/main/package.json#L74
are you sure of the version ?

Comment (oubchid) [reply package.json:34]: hmm I'm using the latest version, let me check if there is no conflict with our webpack version

**Review by rezk2ll — APPROVED** [review]

**Review by tanandy — APPROVED** [review]

**Pull request merged**